tests/test_dwarf.py

```diff
@@ -1723,6 +1723,65 @@ def test_enum(self):
             ),
         )
 
+    def test_enum_typedef(self):
+        prog = dwarf_program(
+            wrap_test_type_dies(
+                (
+                    DwarfDie(
+                        DW_TAG.enumeration_type,
+                        (
+                            DwarfAttrib(DW_AT.name, DW_FORM.string, "color"),
+                            DwarfAttrib(DW_AT.type, DW_FORM.ref4, 1),
+                            DwarfAttrib(DW_AT.byte_size, DW_FORM.data1, 4),
+                        ),
+                        (
+                            DwarfDie(
+                                DW_TAG.enumerator,
+                                (
+                                    DwarfAttrib(DW_AT.name, DW_FORM.string, "RED"),
+                                    DwarfAttrib(DW_AT.const_value, DW_FORM.data1, 0),
+                                ),
+                            ),
+                            DwarfDie(
+                                DW_TAG.enumerator,
+                                (
+                                    DwarfAttrib(DW_AT.name, DW_FORM.string, "GREEN"),
+                                    DwarfAttrib(DW_AT.const_value, DW_FORM.data1, 1),
+                                ),
+                            ),
+                            DwarfDie(
+                                DW_TAG.enumerator,
+                                (
+                                    DwarfAttrib(DW_AT.name, DW_FORM.string, "BLUE"),
+                                    DwarfAttrib(DW_AT.const_value, DW_FORM.data1, 2),
+                                ),
+                            ),
+                        ),
+                    ),
+                    DwarfDie(
+                        DW_TAG.typedef,
+                        (
+                            DwarfAttrib(DW_AT.name, DW_FORM.string, "__uint64_t"),
```

Comment:
This doesn't matter since it's just a test case, but `__uint64_t` being an `unsigned int` doesn't sit right with me :p 

```suggestion
                            DwarfAttrib(DW_AT.name, DW_FORM.string, "__uint32_t"),
```